pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -398,6 +398,10 @@ const (
 	LabelLogging = "logging"
 	// LabelMonitoring is a constant for a label for monitoring stack configurations
 	LabelMonitoring = "monitoring"
+	// LabelKeyCustomLoggingResource is a label key used for the logging custom resources
+	LabelKeyCustomLoggingResource = "fluentbit.gardener/type"
```

Comment:
I can not do this change right now, because the operator will not select the resources which are coming from the extensions: https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/clusterfilters-logging.yaml#L5.

This can be done as a future improvement

Reply:
This is the PR that actually introduces the operator, so we indeed can do it, or am I missing something? There is nothing w.r.t. backwards-compatibility that we would need to take care of, or?

Reply:
Yes, but if we want to do that, I have to first open PRs on all extensions, to wait for these PRs to be merged, to be released, and to be deployed on the landscapes, and right after that this PR can be merged. Do you see the value of doing this and postponing this PR for some weeks?

Reply:
I don't want to hold this PR just because of that as the value is indeed minimal. Though, such "inconsistency" problems can easily go out of hand in the long run, so I suggest to indeed follow up on it eventually. Perhaps it would have been better to first agree on this "API" for extensions by submitting this label and docs as a separate upfront PR here before adapting all extensions. Anyways, too late now. Let's keep it then.